docs/settings/logging-settings.asciidoc

```diff
@@ -4,172 +4,87 @@
-        type: pattern
-        highlight: true
-  root:
-    appenders: [console_appender]
-    level: warn
-  loggers:
-    - name: elasticsearch.query
-      level: debug
---
-
-[[change-overall-log-level]]
-===== Change overall log level
-
-[source,yaml]
-----
-logging:
-  root:
-    level: debug
-----
-
-[[customize-specific-log-records]]
-===== Customize specific log records
-Here is a detailed configuration example that can be used to configure _loggers_, _appenders_ and _layouts_:
-
-[source,yaml]
-----
-logging:
-  appenders:
-    console:
-      type: console
-      layout:
-        type: pattern
-        highlight: true
-    file:
-      type: file
-      fileName: /var/log/kibana.log
-    custom:
-      type: console
-      layout:
-        type: pattern
-        pattern: "[%date][%level] %message"
-    json-file-appender:
-      type: file
-      fileName: /var/log/kibana-json.log
-      layout:
-        type: json
-
-  root:
-    appenders: [console, file]
-    level: error
-
-  loggers:
-    - name: plugins
-      appenders: [custom]
-      level: warn
-    - name: plugins.myPlugin
-      level: info
-    - name: server
-      level: fatal
-    - name: optimize
-      appenders: [console]
-    - name: telemetry
-      appenders: [json-file-appender]
-      level: all
-    - name: metrics.ops
-      appenders: [console]
-      level: debug
-----
-
-Here is what we get with the config above:
-[options="header"]
+The folowing table serves as a quick reference for different logging configuration keys. Note that these are not stand-alone settings and may require additional logging configuration. See the <<logging-configuration, Configure Logging in {kib}>> guide and complete <<log-settings-examples, examples>> for common configuration use cases. 
+
+[cols="2*<"]
 |===
+| `logging.appenders[].<appender-name>`
+| Unique appender identifier.
 
-| Context name | Appenders | Level
+| `logging.appenders[].console:`
+| Appender to use for logging records to *stdout*. By default, uses the `[%date][%level][%logger] %message` **pattern** layout. To use a **json**, set the <<log-in-json-ECS-example,layout type to `json`>>.
 
-| root | console, file | error
+| `logging.appenders[].file:`
+| Allows you to specify a fileName to send log records to on disk. To send <<log-to-file-example,all log records to file>>, add the file appender to `root.appenders`. If configured, you also need to specify <<log-to-file-example, `logging.appenders.file.pathName`>>.
 
-| plugins | custom | warn
+| `logging.appenders[].rolling-file:`
+| Similar to Log4j's `RollingFileAppender`, this appender will log into a file and rotate if following a rolling strategy when the configured policy triggers. There are currently two policies supported: `size-limit` and `time-interval`.
 
-| plugins.myPlugin | custom | info
+| `logging.appenders[].<appender-name>.type`
+| Determines where the log messages are output. Options are `console`, `file`, `rewrite`, `rolling-file`. Required.
```

Comment:
```suggestion
| The appender type determines where the log messages are sent. Options are `console`, `file`, `rewrite`, `rolling-file`. Required.
```

below we talk about "file and rolling-file appender types" so thought using the same language might make it a bit clearer that this is where you choose the type.